<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OwnerWidget</class>
 <widget class="QWidget" name="OwnerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="user-info">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="reloadButton">
       <property name="toolTip">
        <string>Reload data</string>
       </property>
       <property name="text">
        <string>Reload</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editButton">
       <property name="toolTip">
        <string>Edit row</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset theme="tools-check-spelling">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addButton">
       <property name="toolTip">
        <string>Add row</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remButton">
       <property name="toolTip">
        <string>Remove row</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>Quick search</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>reloadButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>remButton</tabstop>
  <tabstop>searchEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>reloadButton</sender>
   <signal>clicked()</signal>
   <receiver>OwnerWidget</receiver>
   <slot>reloadList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>OwnerWidget</receiver>
   <slot>editClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>OwnerWidget</receiver>
   <slot>addClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remButton</sender>
   <signal>clicked()</signal>
   <receiver>OwnerWidget</receiver>
   <slot>remClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reloadList()</slot>
  <slot>editClicked()</slot>
  <slot>addClicked()</slot>
  <slot>remClicked()</slot>
 </slots>
</ui>
